content/en/docs/resources/guides/extending-jx/apps.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Apps
+linktitle: Apps Framework
+description: Extending Jenkins X using the Apps Framework
+type: docs
+weight: 20
+aliases:
+    - /docs/contributing/addons/
+    - /docs/guides/extending-jx/introduction/
+---
+
+## What are Apps
+
+Jenkins X Apps are distributed as Helm Charts via Helm Chart repositories. Any Helm chart can be installed as an app
+using `jx add app`, although Jenkins X adds various capabilities to Helm Charts including:
+
+* the ability to interactively ask questions to generate `values.yaml` based on JSON Schema
+* the ability to create pull requests against the GitOps repo that manages your team/cluster
+* the ability to store secrets in vault
+* the ability to upgrade all apps to the latest version
+
+Planned features include:
+
+* integrating [kustomize](https://github.com/kubernetes-sigs/kustomize) to allow existing charts to be modified
+* storing Helm repository credentials in vault
+* taking existing `values.yaml` as defaults when asking questions based on JSON Schema during app upgrade
+* only asking new questions during app upgrade
+* `jx get apps` - the ability to list all apps that can be installed
```

Comment:
we should remove this point - especially since it is listed among the CLI commands below